pymchelper/utils/runmc.py

```diff
@@ -102,13 +106,15 @@ def main(args=None):
     #   - location of MC simulator executable file (i.e. `shieldhit` or `rfluka`)
     #   - simulation options for the MC engine provided via -m switch (i.e. --time or -v)
     settings = SimulationSettings(input_path=parsed_args.input,
+                                  simulator_type=parsed_args.type,
```

Comment:
There is no such field as `type`.

You have to name param `type` instead `simulator` (line 54).
Alternatively, you can use `dest` as it was done in line 68 ` dest='outtype'`

```suggestion
                                  simulator_type=parsed_args.simulator,
```

Reply:
you're right, I think the field was called "type" earlier and I didn't change it